codegen/cmd/injection-gen/generators/packages.go

```diff
@@ -165,13 +165,26 @@ func MustParseClientGenTags(lines []string) Tags {
 	}
 
 	values := types.ExtractCommentTags("+", lines)
-	// log.Printf("GOT values %v", values)
+
 	_, ret.GenerateDuck = values["genduck"]
 	_, ret.GenerateReconciler = values["genreconciler"]
+	_, ret.GenerateReconciler = values["genreconciler:class"]
```

Comment:
This will overwrite what's above 🤦‍♂

Reply:
yeah that is fine and intended.

I did not want to require `// +genreconciler ` AND `// genreconciler:class=foo.bar`

Reply:
note it is not the value, but the existence of that tag.

Reply:
oooh wait wait... I need an or


fml